backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -97,9 +97,9 @@ internal class RTTIGeneratorVisitor(context: Context) : IrElementVisitorVoid {
 
     override fun visitClass(declaration: IrClass) {
         super.visitClass(declaration)
-
-        generator.generate(declaration)
-
+        if (declaration.requiresRtti()) {
```

Comment:
How do these classes happen to be found in this IR module in the first place?
Shouldn't these classes be among external IR stubs which aren't supposed to go through our lowerings and code generation?

Reply:
Right now external Obj-C classes are usual classes that can be found in the IR module. There are other classes (for example, `CStructVar`) which are defined in the module and should be skipped during RTTI generation (I’ll support them later).